components/Footer/Footer.js

```diff
@@ -8,7 +8,7 @@ import Logo from "../../public/Header img.png";
 const Footer = () => {
   // const bgcolor= "#DADADA"
   return (
-    <footer className=" w-full bg-[#121B24] dark:bg-[#DADADA] min-h-[200px] relative">
+    <footer className="absolute bottom-0 left-0 w-full bg-[#121B24] dark:bg-[#DADADA] min-h-[200px]">
```

Comment:
@AliReza1083  instead of absolute we can use the sticky display to make it stay at the bottom.

Reply:
Position sticky won't fix our problem.

We'll use absolute, and we've set the body to be relative, Whenever the body is large, the footer stick with Body.

Reply:
Got it 👍🏻 thanks for clarifying this 🙂